src/components/User.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+const data = require('../data/data.json')
+
+const users = data
+
+User.propTypes = {
+  userData: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      phone: PropTypes.string.isRequired,
+      mail: PropTypes.string,
+      department: PropTypes.string.isRequired,
+    })
+  ),
+  skills: PropTypes.arrayOf(
+    PropTypes.shape({
+      skill1: PropTypes.string.isRequired,
+      skill2: PropTypes.string,
+      skill3: PropTypes.string,
+      skill4: PropTypes.string,
+      skill5: PropTypes.string,
+      skill6: PropTypes.string,
+    })
+  ),
+}
+
+export default function User() {
+  return (
+    <Wrapper>
+      {users.map(({ id, name, phone, email, department, skills }, index) => (
+        <Details>
+          <li key={index}>{id}</li>
+          <li>Name: {name}</li>
+          <li>Phone: {phone}</li>
+          <li>Mail: {email}</li>
+          <li>Department: {department}</li>
+          <SkillsList>
+            {skills.map((skill, index) => (
+              <div key={index}>{skill}</div>
```

Comment:
I think it would be better to use a `<li>` here

Reply:
Like stated in the comment before: get rid of the other list and only have the skills as a list. And then remember that only list items are valid children of lists.